Sources/Extensions/Shared/ColorExtensions.swift

```diff
@@ -214,8 +214,50 @@ public extension Color {
 		let a = level1*a1 + level2*a2
 		
 		return Color(red: r, green: g, blue: b, alpha: a)
+        
 	}
-	
+    
+    /// SwifterSwift: Lighten a color
+    ///
+    ///     let color = Color(red: r, green: g, blue: b, alpha: a)
+    ///     let lighterColor: Color = color.lighten(by: 0.2)
+    ///
+    /// - Parameter percentage: Percentage by which to lighten the color
+    /// - Returns: A lightened color
+    public func lighten(by percentage: CGFloat = 0.2) -> Color {
+        // https://stackoverflow.com/questions/38435308/swift-get-lighter-and-darker-color-variations-for-a-given-uicolor
+        return self.adjust(by: abs(percentage) )
```

Comment:
@oettingerj We typically don't like SwifterSwift extensions to depend on another SwifterSwift extension.
In this case, `lighten(by:)` and `darken(by:)` both depend on the `adjust(by:)` extension.

We either have to duplicate the internals of `adjust(by:)` or delete both `lighten(by:)` and `darken(by:)`